src/jbuild

```diff
@@ -1,8 +1,12 @@
 (jbuild_version 1)
 
-(executable
- ((name       wsproxy)
-  (public_name wsproxy)
-  (libraries   (re.str lwt lwt.unix))
+(executables
+ ((names (wsproxy wsproxy_tests))
```

Comment:
On adding helpers or base64 as library, jbuilder complains about `Multiple rules generated for _build/default/src/helpers.o`